package.json

```diff
@@ -38,19 +41,25 @@
     "react": "^16.8",
     "react-native": ">=0.59"
   },
+  "dependencies": {
+    "deep-assign": "^3.0.0"
```

Comment:
this package is deprecated a long time ago?

https://www.npmjs.com/package/deep-assign